### PR TITLE
refactor: show warning if API contains optional params only

### PR DESCRIPTION
**Pull request by yuqizhou77**
- show warning if API contains optional params only that we won't generate parameters for the command.

image

### CODE REVIEW

**Review by Alive-Fish — DISMISSED** [review]

**Review by nliu-ms — DISMISSED** [review]

**Review by nliu-ms — DISMISSED** [review]

**Review by nliu-ms — DISMISSED** [review]

File: packages/fx-core/resource/package.nls.json [hunk 1]
```diff
@@ -254,6 +254,8 @@
   "core.copilotPlugin.scaffold.summary.warning.teamsManifest.mitigation": "Mitigation: update \"%s\" field in \"%s\".",
   "core.copilotPlugin.scaffold.summary.warning.teamsManifest.missingCardTemlate": "Missing \"%s\" in command \"%s\".",
   "core.copilotPlugin.scaffold.summary.warning.teamsManifest.missingCardTemlate.mitigation": " Mitigation: create Adaptive Card template in \"%s\" and then update \"%s\" field to the relative path in \"%s\".",
+  "core.copilotPlugin.scaffold.summary.warning.teamsManifest.missingCommandParameters": "Missing parameters in command \"%s\" as we cannot find any required parameter.",
```
Comment (sffamily) [inline packages/fx-core/resource/package.nls.json:257]: as we cannot find any required parameter. -> consider to remove? duplicated with the wording before it.

File: packages/fx-core/resource/package.nls.json [hunk 2]
```diff
@@ -254,6 +254,8 @@
...
+  "core.copilotPlugin.scaffold.summary.warning.teamsManifest.missingCommandParameters.mitigation": " Mitigation: add at least one parameter for command \"%s\" in \"%s\". The parameter name should match what defined in \"%s\".",
```
Comment (sffamily) [inline packages/fx-core/resource/package.nls.json:258]: what defined -> what's defined

File: packages/fx-core/resource/package.nls.json (hunk 1 above)
Comment (yuqizhou77) [reply packages/fx-core/resource/package.nls.json]: updated

File: packages/fx-core/resource/package.nls.json (hunk 2 above)
Comment (yuqizhou77) [reply packages/fx-core/resource/package.nls.json]: updated

**Review by Alive-Fish — APPROVED** [review]

**Review by sffamily — APPROVED** [review]

**Pull request merged**